packages/jazz-tools/src/tools/coValues/coList.ts

```diff
@@ -873,6 +888,15 @@ export class CoListJazzApi<L extends CoList> extends CoValueJazzApi<L> {
   } {
     return (this.coList.constructor as typeof CoList)._schema;
   }
+
+  /**
+   * Deeply merge the current branch into the main CoValues.
+   *
+   * Doesn't have any effect when there are no changes to merge, or the current CoValue is not a branch
+   */
+  unstable_merge() {
+    unstable_mergeBranch(this.coList);
+  }
```

Comment:
Can we move this up to `CoValueJazzApi` to avoid repeating it on every CoValue class?